src/helpers/channels/create_stage_instance.ts

```diff
@@ -0,0 +1,39 @@
+import { validateLength } from "../../util/validate_length.ts";
+import { Errors } from "../../types/discordeno/errors.ts";
+import { rest } from "../../rest/rest.ts";
+import { endpoints } from "../../util/constants.ts";
+import { StageInstance } from "../../types/channels/stage_instance.ts";
```

Comment:
Use import type to import interfaces
```suggestion
import type { StageInstance } from "../../types/channels/stage_instance.ts";
```